### PR TITLE
PREAPPS-4260 || PREAPPS-4652: Support for showing display names for Zimbra X zimlets on admin UI

### PR DESCRIPTION
**Pull request by avadhutprabhudesai**
Changes:
1. Added 'label' attribute to zimlet xml

### CODE REVIEW

File: src/commands/package.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ export default asyncCommand({
 		//Create the xml descriptor file for the zimlet
 		let xmlFile = `${argv.name}.xml`;
 
-		let zimletXML = `<zimlet name="${argv.name}" version="${argv.pkgVersion}" description="${argv.description}" zimbraXZimletCompatibleSemVer="${argv.zimbraXVersion}">`;
+		let zimletXML = `<zimlet name="${argv.name}" version="${argv.pkgVersion}" description="${argv.description}" label="${argv.displayName}" zimbraXZimletCompatibleSemVer="${argv.zimbraXVersion}">`;
```
Comment (silentsakky) [inline src/commands/package.js]: argv.label

Comment (silentsakky) [reply src/commands/package.js]: we should also update help instruction to show that this parameter is supported

Comment (silentsakky) [reply src/commands/package.js]: https://github.com/Zimbra/zimlet-cli/pull/55/files#diff-ee598ee5680e1c35974febfe40adc27fL12

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@zimbra/zimlet-cli",
-  "version": "8.3.0",
+  "version": "8.3.1",
```
Comment (silentsakky) [inline package.json:3]: revert this, in zimlet-cli we don't bump version in PR

Comment (madhavdhuppe) [reply package.json]: done

**Comment by silentsakky** [comment]
Published in @zimbra/zimlet-cli@9.0.0

**Pull request merged**